packages/flutter/lib/src/material/app_bar.dart

```diff
@@ -1023,9 +1026,33 @@ class _AppBarState extends State<AppBar> {
       // a size of 48x48, and a highlight size of 40x40. Users can also put other
       // type of widgets on leading with the original config.
       if (theme.useMaterial3) {
-        leading =  ConstrainedBox(
+        if (leading is IconButton) {
+          // styleFrom method is used to generate a correct overlayColor based on the foregroundColor.
```

Comment:
Thanks for the quick review! Just want to ask if the comment is just for code with two slashes // instead of ///, do we also need brackets for properties and methods?

Reply:
Hmm good point, the guide does specify that only for /// doc comments. I'd say add the [brackets] as a matter of consistency, but I don't have strong feelings - up to you :).

Reply:
I do find some // cases with brackets. Fixed it! Thanks for catching this:)